src/test/java/com/squareup/kotlinpoet/TypeSpecTest.kt

```diff
@@ -510,6 +510,27 @@ class TypeSpecTest {
         |""".trimMargin())
   }
 
+  @Test fun sealedClassesMayDefineAbstractMembers() {
+    val sealedClass = TypeSpec.classBuilder("Sealed")
+        .addModifiers(KModifier.SEALED)
+            .addProperty(PropertySpec.builder("name", ClassName("", "String"))
```

Comment:
Please fix the indentation in this part. Also, `ClassName("", "String")` -> `String::class`